src/transform/mod.rs

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) 2017-2018, The rav1e contributors. All rights reserved
+//
+// This source code is subject to the terms of the BSD 2 Clause License and
+// the Alliance for Open Media Patent License 1.0. If the BSD 2 Clause License
+// was not distributed with this source code in the LICENSE file, you can
+// obtain it at www.aomedia.org/license/software. If the Alliance for Open
+// Media Patent License 1.0 was not distributed with this source code in the
+// PATENTS file, you can obtain it at www.aomedia.org/license/patent.
+
+use partition::TxSize;
+use partition::TxType;
+use partition::TX_TYPES;
+
+use util::*;
+
+use self::forward::*;
+use self::inverse::*;
```

Comment:
```suggestion
pub use self::inverse::*;
```

This should fix the broken tests I guess.